packages/components/src/components/select/select.lite.tsx

```diff
@@ -87,32 +101,23 @@ export default function DBSelect(props: DBSelectProps) {
 	});
 
 	onMount(() => {
-		state._id = props.id || 'select-' + uuid();
+		state._id = props.id || state._id;
 	});
 
 	onUpdate(() => {
 		if (state._id) {
-			state._placeholderId = state._id + DEFAULT_PLACEHOLDER_ID_SUFFIX;
-			state._messageId = state._id + DEFAULT_MESSAGE_ID_SUFFIX;
-			state._validMessageId = state._id + DEFAULT_VALID_MESSAGE_ID_SUFFIX;
-			state._invalidMessageId =
+			const messageId = state._id + DEFAULT_MESSAGE_ID_SUFFIX;
+			const validMessageId = state._id + DEFAULT_VALID_MESSAGE_ID_SUFFIX;
+			const invalidMessageId =
 				state._id + DEFAULT_INVALID_MESSAGE_ID_SUFFIX;
+			const placeholderId = state._id + DEFAULT_PLACEHOLDER_ID_SUFFIX;
+			state._messageId = messageId;
+			state._validMessageId = validMessageId;
+			state._invalidMessageId = invalidMessageId;
+			state._invalidMessageId = placeholderId;
```

Comment:
```suggestion
			state._invalidMessageId = invalidMessageId;
			state._placeholderId = placeholderId;
```

why do you immediately reset this state? looks like a mistake here.